### PR TITLE
Fix fenced code block typo in Buffering middleware page

### PR DESCRIPTION
**Pull request by Wingysam**
### What does this PR do?

Fixes one typo on buffering page in docs

### Motivation

Saw a typo

### More

- [ ] Added/updated tests
- [x] Added/updated documentation

### CODE REVIEW

File: docs/content/middlewares/buffering.md [hunk 1]
```diff
@@ -152,9 +152,9 @@ http:
 
 ### `maxResponseBodyBytes`
 
-With the `maxReesponseBodyBytes` option, you can configure the maximum allowed response size from the service (in Bytes).
+With the `maxResponseBodyBytes` option, you can configure the maximum allowed response size from the service (in Bytes).
 
-If the response exceeds the allowed size, it is not forwarded to the client. The client gets a `413 (Request Entity Too Large) response` instead.
+If the response exceeds the allowed size, it is not forwarded to the client. The client gets a `413 (Request Entity Too Large)` response instead.
```
Comment (tomMoulard) [inline docs/content/middlewares/buffering.md:157]:
```suggestion
If the response exceeds the allowed size, it is not forwarded to the client. The client gets a `413` (Request Entity Too Large) response instead.
```

**Comment by rtribotte** [comment]
Hello @Wingysam,

Thanks for your contribution!

We only accept documentation fixes on the current minor version branch, and on the v2.6 branch, the typo was already fixed.
Hence, we only kept the fix for the backtick position.

**Review by rtribotte — APPROVED** [review]
Thanks 👍

**Comment by Wingysam** [comment]
@rtribotte I hadn't realized that I was reading outdated docs, thanks for fixing it!

**Pull request merged**